umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md

```diff
@@ -35,8 +33,8 @@ handling large amounts of data.
 
 Benthos UMH offers some benefits, including:
 
-- **Management Console integration**: Configure and deploy any number of
-  Benthos UMH instances directly from the Management Console.
+- **Management Console integration**: Configure and deploy any number of protocol converters via
```

Comment:
should we add maybe the full list of what protocols one can read out? that is probably interesting for someone who is browsing through the documentation

Reply:
Full might be rather large? We could list a few, and conclude with "and more" or something like that, as well as link to all available ones.